StandaloneMmPkg/Drivers/StandaloneMmIplPei/StandaloneMmIplPei.c

```diff
@@ -147,12 +147,44 @@ LocateMmCoreFv (
   OUT VOID                  **MmCoreImageAddress
   )
 {
-  EFI_STATUS           Status;
-  UINTN                FvIndex;
-  EFI_PEI_FV_HANDLE    VolumeHandle;
-  EFI_PEI_FILE_HANDLE  FileHandle;
-  EFI_PE32_SECTION     *SectionData;
-  EFI_FV_INFO          VolumeInfo;
+  EFI_STATUS               Status;
+  UINTN                    FvIndex;
+  EFI_PEI_FV_HANDLE        VolumeHandle;
+  EFI_PEI_FILE_HANDLE      FileHandle;
+  EFI_PE32_SECTION         *SectionData;
+  EFI_FV_INFO              VolumeInfo;
+  MM_CORE_FV_LOCATION_PPI  *MmCoreFvLocation;
+
+  //
+  // MmIpl should locate MM Core FV location PPI to find the MM core FV.

```

Comment:
Ray asked to clarify that it's the producer's responsibility that MmCoreFvLocation PPI reports the correct FV that contains MmCore.
And also clarify what's the behavior if not found the gMmCoreFvLocationPpiGuid:

The producer of the MmCoreFvLocation PPI is responsible for ensuring that it reports the correct Firmware Volume (FV) containing the MmCore.
If the gMmCoreFvLocationPpiGuid is not found, the system will search all Firmware Volumes (FVs) to locate the FV that contains the MM Core.

Reply:
updated